### DISCUSSION TITLE
[Bug] User defined route should have higher priority than i18n routes.

### DISCUSSION THREAD

**Discussion by AM-young-fun** [vercel/next.js#49608] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.1.0: Sun Oct  9 20:15:52 PDT 2022; root:xnu-8792.41.9~2/RELEASE_ARM64_T8112
    Binaries:
      Node: 14.21.3
      npm: 6.14.18
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 13.2.4
      eslint-config-next: 13.2.4
      react: 18.2.0
      react-dom: 18.2.0
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Internationalization (i18n), Routing (next/router, next/navigation, next/link)

### Link to the code that reproduces this issue

https://github.com/AntiMoron/next-antd-css.git

### To Reproduce

![image](https://user-images.githubusercontent.com/6587734/229792164-8a3fd3c3-39a9-459b-9c81-71d89ac0bcee.png)

### Describe the Bug

Here're 2 user-defined routes.
- '/jp/a' presenting the word "Japanese"
- '/a' presenting the word "English"

With i18n routing enabled:

```js
{
i18n: {
    locales: ["en", "jp"],
    defaultLocale: "en",
  },
}
```

Every time visit localhost:3000/jp/a always presents "English" which means it always match i18n routes instead user-defined routing.

### Expected Behavior

User-defined routes should have higher priority because in some cases, pages should have different structures.
e.g. Payment methods in Japan differ from those in the U.S. so such pages should have different user interactions. This usually means you have to build up 2 different pages in code.

**Comment by AM-young-fun** [comment]
Is this noticed by any core member?

**Comment by timneutkens** [comment]
The files you've added as `pages/jp/a.tsx` will end up being routed like `/jp/jp/a` for `jp` and `/jp/a` for `en`. In the new App Router we've removed the `i18n` option so that it allows you to use any structure you'd like. You can find the docs here: https://nextjs.org/docs/app/building-your-application/routing/internationalization
